hazelcast-jet-core/src/main/java/com/hazelcast/jet/config/MetricsConfig.java

```diff
@@ -46,9 +46,10 @@
     private int intervalSeconds = DEFAULT_METRICS_COLLECTION_SECONDS;
 
     /**
-     * Sets whether metrics collection should be enabled for the node. If
-     * enabled, Hazelcast Jet Management Center will be able to connect to this
-     * member. It's enabled by default.
+     * Master switch for all metrics related functionality. Enabling it is
```

Comment:
this javadoc is too technical, I would keep the old one and mention that this option is required for other metrics functionality.